README.md

```diff
@@ -16,7 +16,6 @@ received any formal cryptographic and security reviews/audits.
 ## Crates
 | Name         | Crates.io | Documentation | Build Status |
 |--------------|-----------|---------------|--------------|
-| [`aes-ctr`]  | [![crates.io](https://img.shields.io/crates/v/aes-ctr.svg)](https://crates.io/crates/aes-ctr) | [![Documentation](https://docs.rs/aes-ctr/badge.svg)](https://docs.rs/aes-ctr) | ![build](https://github.com/RustCrypto/stream-ciphers/workflows/aes-ctr/badge.svg?branch=master&event=push)
```

Comment:
Note that this PR deletes the `aes-ctr` crate as its functionality has been subsumed into the `aes` crate.